### PR TITLE
Update AWS SDK to 1.10.10 

### PR DESCRIPTION
**Pull request by Fsero**
cloned PR from https://github.com/elastic/elasticsearch-cloud-aws/pull/232/files

### CODE REVIEW

**Comment by dadoonet** [comment]
The `pom.xml` change is missing, right?

**Comment by Fsero** [comment]
is already on 1.10.10 thats because i didnt change it see on https://github.com/elastic/elasticsearch/blob/master/plugins/cloud-aws/pom.xml#L19

**Comment by dadoonet** [comment]
It's `1.10.0` right?

**Comment by Fsero** [comment]
Ohh!! totally sorry about that :)

File: plugins/cloud-aws/src/test/java/org/elasticsearch/cloud/aws/AmazonS3Wrapper.java [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
- * the Apache License, Version 2.0 (the "License"); you may
- * not use this file except in compliance with the License.
- * You may obtain a copy of the License at
+ * Licensed to Elasticsearch  under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. Author licenses this
+ * file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
```
Comment (dadoonet) [inline plugins/cloud-aws/src/test/java/org/elasticsearch/cloud/aws/AmazonS3Wrapper.java]: Can you keep the License header as it was?

**Comment by Fsero** [comment]
done.

**Comment by daviddyball** [comment]
Will this change get back-ported to the 1.4 Docker containers or will it only come to 1.7+ versions of ES? Currently the errors in elastic/elasticsearch-cloud-aws#233 are preventing us from implementing correct backup/restore functionality with an S3 backend and we can't progress until this is fixed.

**Comment by FestivalBobcats** [comment]
I'm curious as well how to patch ES 1.4.x - 1.7.0. I've applied the changes here in the 1.4 branch but tests are failing.

**Comment by jhansen-tt** [comment]
Please release 1.7.2 ASAP with this fix.  This is a major issue.

**Comment by dadoonet** [comment]
@jhansen-tt feel Free to test the SNAPSHOT version and report in this PR https://github.com/elastic/elasticsearch-cloud-aws/pull/234#issuecomment-134171132

**Comment by jhansen-tt** [comment]
Is there a SNAPSHOT PPA?

**Comment by dadoonet** [comment]
Just follow the instructions in the PR comment.
Thanks!

**Pull request closed without merging**